DatadogInternal/Sources/NetworkInstrumentation/URLSession/URLSessionTask+Tracking.swift

```diff
@@ -33,4 +33,20 @@ extension DatadogExtension where ExtendedType: URLSessionTask {
 
         return session.delegate
     }
+
+    var hasCompletion: Bool {
+        get {
+            let value = objc_getAssociatedObject(type, &hasCompletionKey) as? Bool
+            return value == true
+        }
+        set {
+            if newValue {
+                objc_setAssociatedObject(type, &hasCompletionKey, true, .OBJC_ASSOCIATION_RETAIN_NONATOMIC)
```

Comment:
blocking/ We rather mean following, no?
```suggestion
                objc_setAssociatedObject(type, &hasCompletionKey, newValue, .OBJC_ASSOCIATION_RETAIN_NONATOMIC)
```

Reply:
not blocking, if check the `if block`, if the newValue is false, we remove it.

Reply:
Oh, I've read it as `if let value`! Forget it 🙂👍